azerothcore/azerothcore-wotlk
# fix(Core/SpellScript): "Egg nog" item now transforms you

## CHANGES PROPOSED:
-  Egg Nob - Script Item

https://classic.wowhead.com/item=17198/egg-nog#guides

you should have a chance to be transformed into [Mosshoof Runner] or [Strange Snowman]

According to "data" one of this 2 spells could be triggered upon 21149 is cast:
https://classic.wowhead.com/spell=21936/reindeer
https://classic.wowhead.com/spell=21980/snowman

https://wowwiki.fandom.com/wiki/Egg_Nog

## ISSUES ADDRESSED:
- Closes 



## TESTS PERFORMED:
-
-



## HOW TO TEST THE CHANGES:

- `.additem 17198`
- use the item and see the chances of transforming [Mosshoof Runner] or [Strange Snowman]

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
